elements/pl-dropdown/pl-dropdown.mustache

```diff
@@ -0,0 +1,36 @@
+{{#question}}
+    <div id="pl-dropdown-{{uuid}}" class="pl-dropdown form-group">
+        <select name="{{correct-answer}}" class="form-control">
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+
+    {{#options}}
+        <option value="{{.}}">{{.}}</option>
+    {{/options}}
+
+        </select>
+    </div>
+{{/question}}
+
+{{#submission}}
+    {{#no-answer-selected}}
+        <span class="badge pl-dropdown-no-answer">&lt;No answer selected&gt;</span>
+    {{/no-answer-selected}}
+    {{^no-answer-selected}}
+        <span class="badge pl-dropdown-submitted">{{submitted-answer}}</span>
```

Comment:
```suggestion
        <span class="badge bg-warning d-inline">{{submitted-answer}}</span>
```

Reply:
This also makes sense. Just double-checked the colours match-up. Thanks!